tests/jvm/src/test/resources/lowlevel.expect

```diff
@@ -39,7 +114,7 @@ _root_.scala.Unit# => abstract final class Unit.{+2 decls}
   extends AnyVal
 _root_.scala.collection. => package collection
 _root_.scala.collection.mutable. => package mutable
-_root_.scala.collection.mutable.Stack# => class Stack.{+20 decls}
+_root_.scala.collection.mutable.Stack# => @deprecated @deprecated class Stack.{+20 decls}
```

Comment:
Why `@deprecated` twice?

Reply:
You wouldn't believe: https://github.com/scala/scala/blob/c2a5883891a68180b143eb462c8b0cebc8d3b021/src/compiler/scala/tools/nsc/symtab/classfile/ClassfileParser.scala#L828-L831.